docs/config/README.md

```diff
@@ -1179,6 +1180,37 @@ The module will be shown if any of the following conditions are met:
 symbol = "🐪 "
 ```
 
+## Perl
+
+The `perl` module shows the currently installed version of Perl.
+The module will be shown if any of the following conditions are met:
+
+- The current directory contains a `Makefile.PL` file
+- The current directory contains a `cpanfile` file
+- The current directory contains a `META.json` file
+- The current directory contains a `META.yml` file
+- The current directory contains a `.perl-version` file
+- The current directory contains a `.pl` file
+- The current directory contains a `.pm` file
+
+### Options
+
+| Variable   | Default      | Description                                           |
+| ---------- | ------------ | ----------------------------------------------------- |
+| `symbol`   | `"🐪 "`      | The symbol used before displaying the version of PHP. |
+| `style`    | `"bold 149"` | The style for the module.                             |
+| `disabled` | `false`      | Disables the `perl` module.                           |
+
+### Example
+
+```toml
+# ~/.config/starship.toml
+
+[perl]
+symbol = "🐫 "
```

Comment:
Nasty, I'm sure OCaml folks would loudly object to the abuse of their camel 😀

Reply:
this was a joke, as just the same thing is done a few lines up, i replace example by an oyster

Reply:
My comment was also a joke 🤣